.github/workflows/GH-R-CMD-check.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
           try(pak::pkg_install("tidymodels/tune#804"))
           try(pak::pkg_install("tidymodels/workflows"))
           try(pak::pkg_install("tidymodels/yardstick"))
-          try(pak::pkg_install("tidymodels/finetune"))
+          try(pak::pkg_install("tidymodels/finetune#94"))
```

Comment:
Wasn't able to correct this ref before merge; will put in a follow-up PR in a moment.

Reply:
For reference: https://github.com/tidymodels/extratests?tab=readme-ov-file#pr-pairs-with-package-repos